leasing/management/commands/set_group_field_permissions.py

```diff
@@ -499,7 +502,26 @@
             6: "view",
             7: "change",
         },
+        "old_dwellings_in_housing_companies_price_index": {
+            1: "view",
+            2: "view",
+            3: "view",
+            4: "view",
```

Comment:
This must be change. The same for old_dwellings_in_housing_companies_price_index_type below
PR to be merged here: https://github.com/City-of-Helsinki/mvj/pull/804